app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -478,6 +516,46 @@ class VaultAddEditViewModel @Inject constructor(
         }
     }
 
+    private fun handleUserVerificationLockOut() {
+        showFido2ErrorDialog()
+    }
```

Comment:
🎨 I guess this is to follow same structure as in other places but couldn't we directly call `showFido2ErrorDialog()` when `VaultAddEditAction.Common.UserVerificationLockOut` (also applying to `VaultAddEditAction.Common.UserVerificationFail`)? 

IMO it'd be easier to follow and we avoid having a func that don't do anything but to call another func which per transitivity we can call directly in the first place without losing context given that the name of the func is not adding anything of what we already know in the `when`. Moreover, we end up having a lot of boilerplate code that IMO doesn't add much value.

So I think that these handling functions should only be added when there is more to be done than a simple call to another function but I'd love to hear your thoughts and @david-livefront on this.

Reply:
From a maintenance perspective, I've actually found this convention pretty helpful. I feel like having a mix of functions and inline calls makes locating the handling code more difficult. If I want to know how CloseClick is handled I can easily locate the logic by finding `handleCloseClick` in the corresponding view model because I know that's the established convention. Without that convention in place I have to search for references of `*Action.CloseClick` and filter out irrelevant ones.